src/App/Pages/Vault/BaseCipherViewModel.cs

```diff
@@ -12,18 +12,13 @@ namespace Bit.App.Pages
 {
     public abstract class BaseCipherViewModel : BaseViewModel
     {
-        private CipherView _cipher;
         private readonly IAuditService _auditService;
         protected readonly IDeviceActionService _deviceActionService;
         protected readonly ILogger _logger;
         protected readonly IPlatformUtilsService _platformUtilsService;
+        private CipherView _cipher;
         protected abstract string[] AdditionalPropertiesToRaiseOnCipherChanged { get; }
 
-        public CipherView Cipher
-        {
-            get => _cipher;
-            set => SetProperty(ref _cipher, value, additionalPropertyNames: AdditionalPropertiesToRaiseOnCipherChanged);
-        }
         public AsyncCommand CheckPasswordCommand { get; }
```

Comment:
This is also a property so it has to be below constructor, and we tend to put them below other properties => so it will be below Cipher